cli/src/test/java/de/jplag/cli/MergingOptionsTest.java

```diff
@@ -0,0 +1,19 @@
+package de.jplag.cli;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.api.Test;
+
+import de.jplag.merging.MergingOptions;
+
+public class MergingOptionsTest extends CommandLineInterfaceTest {
```

Comment:
Add a short JDoc comment here. Along the lines of "Test cases for the options of the match merging mechanism."